interactions/client.py

```diff
@@ -148,18 +150,17 @@ def decorator(coro: Coroutine) -> Any:
                     self.me = User(**data)
 
                 request = self.http.create_application_command(
-                    self.me._json["id"], data=_payload, guild_id=guild
+                    self.me.id, data=_payload, guild_id=guild
                 )
 
                 _att = self.loop.run_until_complete(request)
-                print(f"request returns...: {_att}")
 
-                if _att["code"] == 50035:
-                    raise JSONException(50035)  # todo: work on this pls
+                if _att.get("code"):
```

Comment:
Have you tested this in the event that it produces back a 2XX event that is not `200 OK`? We want to comfortably show exception codes that only apply outside of that ruleset.

Reply:
When it succeeded, it didn't return a code, just the object in question, which was weird.

Reply:
This is mildly concerning. What if we throw a warning/info log on 2XX? That might be nice for some bot devs.

Reply:
We'll make this another thing sep. to this PR for commit.